src/Apps/MyCollection/Routes/EditArtwork/Components/MyCollectionArtworkFormImages.tsx

```diff
@@ -97,6 +157,7 @@ export const MyCollectionArtworkFormImages: React.FC<MyCollectionArtworkFormImag
         "newPhotos",
         values.newPhotos.filter(p => p.id !== photo.id)
       )
+      setlocalImages(localImages.filter(p => p.photoID !== photo.id))
```

Comment:
**question:** I guess, this function call is removing the image from the local storage. Maybe we can make it more explicit by adding a comment or extracting the functionality into a function.

Reply:
that's true - i will add a comment